### ISSUE TITLE
TTWG Meeting 2020-10-22

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/ttwg#154]
Agenda to follow.

**Comment by gkatsev** [comment]
agenda+ process2020 and potential for making webvtt a living standard.

**Comment by gkatsev** [comment]
Actually, may miss this due to the chairs meeting (it overlaps the first half hour) and other reasons.

**Comment by nigelmegitt** [comment]
Apologies for the late regrets.

@gkatsev suggest we push process2020 discussion back to next time you can make it.

**Comment by gkatsev** [comment]
Yeah, makes sense to shift it to the next meeting. Hopefully the Chairs meeting will have some info about it as well.

**Comment by cconcolato** [comment]
If possible, can we add to the agenda an incoming liaison letter from MPEG regarding ISO/IEC 14496-30 (carriage of TTML in MP4)? I doubt it will be received by then but I can present what was sent.

**Comment by nigelmegitt** [comment]
Hi @cconcolato I think today's meeting is not going ahead due to the lack of Chair attendance. Sorry for the late notice. I'm on vacation all week and haven't been able to give this meeting the attention needed. 

Can we add it to the agenda for the next meeting?
